src/snowflake/snowpark/functions.py

```diff
@@ -2814,12 +2814,15 @@ def lead(
     )
 
 
-def ntile(e: ColumnOrName) -> Column:
+def ntile(e: Union[int, ColumnOrName]) -> Column:
```

Comment:
I just add `int` to avoid breaking change. Should we deprecate using column? @snowflakedb/snowpark-python-api

Reply:
Yeah we probably should. Let's add a warning first? Then in two releases we could use a BCR to remove it.

Reply:
It's a sql documentation problem since sql function ntile actually works when a column expression is used.
So I suggest we shouldn't deprecate it.

Reply:
Oh I'm not aware of that. If column works then we should just keep the current way.